packages/test-utils/src/variables.ts

```diff
@@ -19,3 +19,8 @@ export const TEST_SIWE_PARAMS = {
   domain: 'localhost',
   uri: 'http://localhost:3000',
 };
+
+export const TEST_ECDSA_PUBLIC_KEY =
```

Comment:
Is it possible to provided the public key without the PEM header and footer i.e. just use the base64? On the server-side we can do some adjustment if needed: add it back, OR not bother if the library allows non-PEM format, or ...

(cc @cygnusv )